src/bulk/common.ts

```diff
@@ -1143,14 +1148,7 @@ class BulkOperationBase {
     return Promise.reject(err);
   }
 
-  /**
-   * An internal helper method. Do not invoke directly. Will be going away in the future
-   *
-   * @function
-   * @param _writeConcern
-   * @param options
-   * @param callback
-   */
+  /** An internal helper method. Do not invoke directly. Will be going away in the future */
   bulkExecute(
     _writeConcern?: WriteConcern,
     options?: object,
```

Comment:
Do we have an interface for this options?

Reply:
I fixed this but also realized we needed to make some changes to `isPromiseLike`.

Reply:
Trying to mess with `isPromiseLike` it's still not correct.